dist/leaflet.css

```diff
@@ -52,10 +52,11 @@
 	max-height: none !important;
 	width: auto;
 	padding: 0;
+	}
+.leaflet-container img.leaflet-tile {
 	/* See: https://bugs.chromium.org/p/chromium/issues/detail?id=600120 */
 	mix-blend-mode: plus-lighter;
-
-	}
+}

```

Comment:
Nitpicking: This closing brace should be indented, for consistency with the rest.